pennylane_pq/ops.py

```diff
@@ -65,7 +65,7 @@ class T(Operation): #pylint: disable=invalid-name,too-few-public-methods
     num_wires = 1
     par_domain = None
 
-class SqrtX(Operation):
+class SqrtX(plops.Operation):
     r"""Square root X gate.
 
     .. math:: SqrtX = \begin{bmatrix}1+i&1-i\\1-i&1+i\end{bmatrix}
```

Comment:
I guess that if `SqrtSwap` is written as `\mathrm{SqrtSwap}`, then `SqrtX` should also be, for consistency. I think that looks better as well for names longer than one character.
```suggestion
    .. math:: \mathrm{SqrtX} = \begin{bmatrix}1+i&1-i\\1-i&1+i\end{bmatrix}
```